src/main/java/com/parasoft/findings/jenkins/coverage/ParasoftCoverageReportScanner.java

```diff
@@ -85,7 +85,7 @@ protected Optional<ProcessedFileResult> processFile(Path file, Charset charset,
             String coberturaPattern = StringUtils.replace(PATH_UTIL.getRelativePath(Paths.get(workspaceLoc),
                     outputCoberturaReport), StringUtils.SPACE, QUESTION_MARK);
             return Optional.of(new ProcessedFileResult(coberturaPattern, generatedCoverageBuildDir.toString()));
-        } catch (Exception exception) {
+        } catch (Exception exception) { // parasoft-suppress OWASP2021.A5.NCE "Reviewed"
```

Comment:
This is expected. - "Do not fail the build when processing coverage reports"

Reply:
Please add the information to the comments to make it clearer. This way, every time we read the suppression, we will understand the reason for catching a general exception instead of throwing it. Otherwise, it may appear risky not to throw it and suppress the violation.